docs/pages/web/pulsar.js

```diff
@@ -2,17 +2,26 @@
 import { type Node } from 'react';
 import AccessibilitySection from '../../docs-components/AccessibilitySection.js';
 import docgen, { type DocGen } from '../../docs-components/docgen.js';
+import doHighlight from '../../examples/pulsar/doHighlight.js';
+import dontMultiple from '../../examples/pulsar/dontMultiple.js';
+import dontPopover from '../../examples/pulsar/dontPopover.js';
+import dontPosition from '../../examples/pulsar/dontPosition.js';
+import dontStatus from '../../examples/pulsar/dontStatus.js';
+import doPopover from '../../examples/pulsar/doPopover.js';
+import doPosition from '../../examples/pulsar/doPosition.js';
+import doPrioritize from '../../examples/pulsar/doPrioritize.js';
 import GeneratedPropTable from '../../docs-components/GeneratedPropTable.js';
 import main from '../../examples/pulsar/main.js';
 import MainSection from '../../docs-components/MainSection.js';
 import Page from '../../docs-components/Page.js';
 import PageHeader from '../../docs-components/PageHeader.js';
 import paused from '../../examples/pulsar/paused.js';
-import positioning from '../../examples/pulsar/positioning.js';
 import QualityChecklist from '../../docs-components/QualityChecklist.js';
 import SandpackExample from '../../docs-components/SandpackExample.js';
 import size from '../../examples/pulsar/size.js';
 
+const bestPracticesHeight = 600;
```

Comment:
I don;t think we need 600  height unless there's some positioning issue that forces us to it. I'd go lower